packages/mdc-list/constants.ts

```diff
@@ -45,6 +45,8 @@ const strings = {
     .${cssClasses.LIST_ITEM_CLASS} button:not(:disabled),
     .${cssClasses.LIST_ITEM_CLASS} a
   `,
+  ENABLED_CHECKBOX_RADIO_SELECTOR: 'input[type="checkbox"], input[type="radio"]',
```

Comment:
I think `ENABLED_` prefix should be added to above `CHECKBOX_RADIO_SELECTOR` & `CHECKBOX_SELECTOR` instead since these selectors also selects disabled controls.

Reply:
Agreed